app/_includes/landing_pages/entity_card.md

```diff
@@ -0,0 +1,14 @@
+{% assign entity_id = include.config.entity %}
+
+{% assign entities = site.collections | first %}
+{% assign entity = entities.docs | where_exp: "doc", "doc.entities contains entity_id" | first %}
```

Comment:
this is great!
you can actually call collections by their name, e.g. `site.kong_entities`.

Reply:
weird, we are using it [here](https://github.com/Kong/developer.konghq.com/blob/main/app/kong-entities.html#L7).
which part isn't working for you? the `where_exp` or fetching the collection?

Reply:
I tried again and it works now, thanks! I think originally I may have only tried `site.collections.kong_entities` 😅